artemis/src/main/java/tech/pegasys/artemis/cli/options/DataOptions.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.cli.options;
+
+import picocli.CommandLine;
+
+public class DataOptions {
+
+  public static final String DATA_PATH_OPTION_NAME = "--data-path";
+  public static final String DATA_STORAGE_MODE_OPTION_NAME = "--data-storage-mode";
+
+  public static final String DEFAULT_DATA_PATH = ".";
+  public static final String DEFAULT_DATA_STORAGE_MODE = "prune";
+
+  @CommandLine.Option(
+      names = {DATA_PATH_OPTION_NAME},
+      paramLabel = "<FILENAME>",
+      description = "Path to output data files",
+      arity = "1")
+  private String dataPath = DEFAULT_DATA_PATH;
+
+  @CommandLine.Option(
+      names = {DATA_STORAGE_MODE_OPTION_NAME},
+      paramLabel = "<STORAGE_MODE>",
+      description =
+          "Sets the strategy for handling historical chain state.  Supported values include: 'prune', and 'archive'",
+      arity = "1")
+  private String dataStorageMode = DEFAULT_DATA_STORAGE_MODE;
```

Comment:
Would be nice to [use enums](https://github.com/PegaSysEng/teku/blob/a99f47058d37d21f57567b632a9cffe9c03e3ddc/storage/src/main/java/tech/pegasys/artemis/storage/StateStorageMode.java#L18) where possible similar to how we [handle enums in besu](https://github.com/mbaxter/besu/blob/4fa78d61a3571a098ecbaa0c9980f7475ae7ca7a/besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java#L327-L332).  But its a bit off-topic, so fine to leave that for a future refactor (not necessarily completed by you).

Reply:
that is on the list of follow up items :))